docs/introduction/get-started/create-account-use-faucet.md

```diff
@@ -80,7 +80,7 @@ Save the account ID for a future step.
 1. Import the note that you have received using the following commands: 
 
       ```shell
-      miden-client input-notes -i <path-to-note>/note.mno
+      miden input-notes -i <path-to-note>/note.mno
```

Comment:
`input-notes` has been renamed to `notes`. Also, `import` has been moved into a top level command (as well as export)

Reply:
also, now that you can mint a public note, should we change the tutorial so we use that instead of downloading the file (or mention both ways, with a private and a public note)

Reply:
Good point, changed both.